lib/src/operations/cardano_derive_native_script_hash_operation.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:typed_data';
+import 'package:ledger_cardano/src/models/parsed_native_script.dart';
+import 'package:ledger_cardano/src/operations/cardano_ledger_operation.dart';
+import 'package:ledger_cardano/src/utils/hex_utils.dart';
+import 'package:ledger_cardano/src/utils/serialization_utils.dart';
+import 'package:ledger_flutter/ledger_flutter.dart';
+
+class CardanoDeriveNativeScriptHashOperation
```

Comment:
This is `work in progress` right? I won't review this file

Reply:
yes